frontend/src/components/jobs/result/visualizer/validators.js

```diff
@@ -112,6 +112,17 @@ function parseElementListOfDict(rawElementList) {
   });
 }
 
+// parse list of Column Elements
+function parseColumnElementList(rawElementList) {
+  return {
+    name: parseString(rawElementList.name),
+    maxWidth: rawElementList.max_width,
```

Comment:
I suggest to add a validation method here: we have a validation function also for the element size, in this way we are "compliant" to the existing logic.